ui/protractor-saucelabs.conf.js

```diff
@@ -0,0 +1,73 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+const { SpecReporter } = require('jasmine-spec-reporter');
+
+exports.config = {
+  jasmineNodeOpts: {
+    defaultTimeoutInterval: 5000000
+  },
+  sauceUser: process.env.SAUCE_USERNAME,
+  sauceKey: process.env.SAUCE_ACCESS_KEY,
+  multiCapabilities: [
```

Comment:
We should define what are the default targets (IE/Safari/Chrome/... and the OS).

Reply:
I created a follow-up issue for @sabbyanandan to define:

https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/720